app_capabilities.go

```diff
@@ -53,20 +58,24 @@ func (e AppCapabilities) Get(stackId, appId, capId int64) (*types.Capability, er
 }
 
 // Create - POST /orgs/:orgName/stacks/:stackId/apps/:app_id/capabilities
-func (e AppCapabilities) Create(stackId, appId int64, capability *types.Capability) (*types.Capability, error) {
-	rawPayload, _ := json.Marshal(capability)
+func (e AppCapabilities) Create(stackId, appId int64, capabilities []*types.Capability, blocks []*types.Block) ([]*types.Capability, error) {
+	input := CreateCapabilitiesInput{
```

Comment:
It looks like this function was never implemented correctly.
Am I reading this correctly?

Reply:
The functionality changed over time and we never updated the go-api-client to match.